jekyll/_cci2/gh-bb-integration.md

```diff
@@ -86,24 +86,51 @@ ERROR: Repository not found.
 Permission denied (publickey).
 ```
 
+## Machine User Keys
+
+For security reasons,
```

Comment:
I think we could be more specific here to be clear about what this prevents (losing credentials when they are linked to a dedicated/single user)

Reply:
Yep, makes sense to me! The delicate balance I'm trying to strike here is between being too vague and restating what GitHub says about machine users. ⚖️